tests/test_note_comments.py

```
@@ -55,21 +55,21 @@ def one(tag):
     return one("w:commentRangeStart"), one("w:commentRangeEnd")
 
 
+_MARKER_TAGS = ("w:commentRangeStart", "w:commentRangeEnd", "w:commentReference")
+
+
 def _marker_order(doc):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `_marker_order`

(ANN202)